src/main/java/com/example/books/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,20 @@
 package com.example.books.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
     private String title;
+    @NotNull
     private String author;
+    @NotNull
     private String isbn;
+    @NotNull
+    @Min(0)
     private BigDecimal price;
     private String description;

```

Comment:
Validate description

Reply:
![image](https://github.com/DZarudnyi/SpringBootBookShop/assets/77063748/b8ad21f0-7601-42af-86b7-9e4006c0b652)
What exactly are you expecting to validate? A string field without constraints that can be null?